db/version_set.cc

```diff
@@ -2833,6 +2833,8 @@ Status Version::MultiGetAsync(
         std::vector<Status> statuses = folly::coro::blockingWait(
             folly::coro::collectAllRange(std::move(mget_tasks))
                 .scheduleOn(&range->context()->executor()));
+        assert(mget_tasks.size() == 0);
```

Comment:
I assume this assertion should be removed?

Reply:
Yes. It looks like there's no value in the assertion, since the remedy is to just clear the vector.